main/modes/games/cGrove/Sparring/cg_Spar.h

```diff
@@ -2,7 +2,7 @@
  * @file cg_spar.h
  * @author Jeremy Stintzcum (jeremy.stintzcum@gmail.com)
  * @brief Provides the sparring implementation for Chowa Grove
- * @version 0.1
+ * @version 0.5.0
```

Comment:
Version 1.0? Or do you individually version files?

Reply:
Individually, 'cuz this is missing a lot of what I wanted. It's not like the version numbers mean much with the haphazard way I use them anyway.